src/rpc/rawtransaction.cpp

```diff
@@ -37,11 +37,11 @@
 
 #include <univalue.h>
 
-/** High fee for sendrawtransaction and testmempoolaccept.
- * By default, transaction with a fee higher than this will be rejected by the
- * RPCs. This can be overridden with the maxfeerate argument.
+/** High fee rate for sendrawtransaction and testmempoolaccept.
+ * By default, transaction with a fee rate higher than this will be rejected by
+ * the RPCs. This can be overridden with the maxfeerate argument.
  */
-constexpr static CAmount DEFAULT_MAX_RAW_TX_FEE{COIN / 10};
+constexpr static CAmount DEFAULT_MAX_RAW_TX_FEE_RATE{COIN / 10};
```

Comment:
Can the type be changed as well?

```suggestion
constexpr static CFeeRate DEFAULT_MAX_RAW_TX_FEE_RATE{COIN / 10};
```

Reply:
`CFeeRate` can't be `constexpr` since it doesn't have `constexpr` constructor.
We can change it to `const` though.

Reply:
Yeah,`static const` is more than enough for our purposes

Reply:
@Remagpie Mind to address this? I think changing the type makes it harder to get wrong again in the code in the future.

Reply:
Sorry for the late response. I think there's no problem if it's okay to use `static const` here.
I'll fix this.